cves/kernel/CVE-2014-0196.yml

```diff
@@ -55,7 +55,7 @@ description_instructions: |
 
   Your target audience is people just like you before you took any course in
   security
-description:
+description: the n_tty_write function in the linux kernel does not properly manage tty driver access in the LECHO & !OPOST case. This will allow users to cause a race condition that will result in an overflow that can grant them priviledges to the system or cause a denial of service using long strings.
```

Comment:
This might be a little to specific with the function names and variables. Otherwise the description was pretty good.